src/main/java/com/github/manolo8/darkbot/core/manager/MapManager.java

```diff
@@ -469,6 +472,7 @@ public boolean setTarget(long entity) {
     }
 
     public boolean isOutOfMap(double x, double y) {
+        if (main.config.GENERAL.ROAMING.RADIATION || radiation.getValue()) return false;
         return x < 0 || y < 0 || x > internalWidth || y > internalHeight;
```

Comment:
This code is checking for the value of the config... twice

```suggestion
        return avoidRadiation.getValue() && (x < 0 || y < 0 || x > internalWidth || y > internalHeight);
```

Reply:
the reason is that I dont know which one should be prioritized. I mean which way of getting the value

Reply:
Inside bot code, for just reading, either is fine. If read & write are used then it should be done via the ConfigSetting api. Regardless, i think new api is cleaner as you define it once then use it as a field of your own class.